<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wikiLookup</class>
 <widget class="QDialog" name="wikiLookup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="wikiLabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NDISWrapper Wiki</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="searchLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>search : </string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="searchEdit"/>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="searchButton">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>lookup</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QPlainTextEdit" name="outputWindow"/>
   </item>
   <item row="2" column="0" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>453</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="cancelButton">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>close Window</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="installButton">
     <property name="text">
      <string>continue installation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>wikiLookup</receiver>
   <slot>slot_searchButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>installButton</sender>
   <signal>clicked()</signal>
   <receiver>wikiLookup</receiver>
   <slot>slot_installButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>wikiLookup</receiver>
   <slot>slot_cancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_cancelButtonClicked()</slot>
  <slot>slot_installButtonClicked()</slot>
  <slot>slot_searchButtonClicked()</slot>
 </slots>
</ui>
